news/news.md

```diff
@@ -1,5 +1,6 @@
 # News
 ## 09 August 2017
+* [Bitcoin Cash](bitcoin_cash_09-08-17.md)
```

Comment:
Please, give a more descriptive subject.  Like "BCC difficulty is down, but mining BTC is still more profitable"